peer_mentoring/chat/views.py

```diff
@@ -3,11 +3,20 @@
 from django.shortcuts import render
 from groups.models import Group
 
+from .models import ChatMessage
+
 
 @login_required
 def group_chat_room(request, group_id):
     group = Group.objects.get(pk=group_id)
+    chat_messages = ChatMessage.objects.filter(group=group)
```

Comment:
TIL, don't pass in "messages" as variable into the template, like `requests` it's a special variable in Django (messages framework)